pl_bolts/models/regression/logistic_regression.py

```diff
@@ -77,8 +77,8 @@ def training_step(self, batch, batch_idx):
     def validation_step(self, batch, batch_idx):
         x, y = batch
         x = x.view(x.size(0), -1)
-        y_hat = self(x)
-        acc = accuracy(y_hat, y)
+        y_hat = self.linear(x)
+        acc = accuracy(F.softmax(y_hat, -1), y)
```

Comment:
but this is in the `forward`, right?

Reply:
I'm not sure I understand the question...
#655 has `training_step()` call `self.linear(x)` so have made `validation_step()` and `test_step()` do likewise.

An alternative approach would be to remove softmax from `forward()` and have it return raw logits